packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -71,6 +71,8 @@ export enum EmailEventStatusEnum {
   BOUNCED = 'bounced',
   DROPPED = 'dropped',
   CLICKED = 'clicked',
+  SPAM_COMPLAINED = 'spam complained',
+  SUBSCRIPTION_CHANGED = 'subscription changed',
```

Comment:
@scopsy @davidsoderberg Seeing how this enum is evolving I insist in my proposal of us setting the internal statuses we need and having separate enums for the provider statuses and map from external to internal, to avoid this enum to become unusable.

Reply:
This makes sense, the only thing we would need to figure out in this case, is what is the common level of enum we support. Because some providers are more verbose than others. So we would need to find a common ground for all of them.